components/CollectionPictureList.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image'
+import React from 'react'
+import PictureComment from './helpers/PictureComment'
+
+export default function CollectionPictureList({ image }) {
```

Comment:
please add an interface for the props of this component

Reply:
done